packages/manager/modules/pci/src/projects/project/data-processing/submit-job/submit-job.scss

```diff
@@ -0,0 +1,16 @@
+.spinner__label {
```

Comment:
You shouldn't need this file, why do you need it ?

Reply:
I can't find a way to have a oui-icon vertically aligned with following text, idem for the spinner. Is there a way to achieve that with existing classes?